contrib/kafka/config/SECURITY.md

```diff
@@ -0,0 +1,38 @@
+# Security
+
+kafka channel controller and dispatcher connect to brokers without
+encryption and authentication by default. Encryption using SSL can be
+turned.
```

Comment:
This isn't quite a complete sentence. You might also want to capitalize "Kafka" on line 3.